.github/workflows/ci.yml

```diff
@@ -0,0 +1,79 @@
+# Used as inspiration: https://github.com/mvdan/github-actions-golang
+
+name: Tests
+
+on:
+  push:
+  pull_request:
```

Comment:
I think it's better to specify the branch name, because otherwise it runs twice on PR pushes, iirc.

Reply:
done